code/datums/components/drift.dm

```diff
@@ -32,6 +32,7 @@
 	RegisterSignal(drifting_loop, COMSIG_MOVELOOP_STOP, .proc/drifting_stop)
 	RegisterSignal(drifting_loop, COMSIG_MOVELOOP_PREPROCESS_CHECK, .proc/before_move)
 	RegisterSignal(drifting_loop, COMSIG_MOVELOOP_POSTPROCESS, .proc/after_move)
+	RegisterSignal(movable_parent, COMSIG_MOVABLE_NEWTONIAN_MOVE, .proc/newtonian_impulse)
 	RegisterSignal(drifting_loop, COMSIG_PARENT_QDELETING, .proc/loop_death)
```

Comment:
```suggestion
	RegisterSignal(drifting_loop, COMSIG_PARENT_QDELETING, .proc/loop_death)
	RegisterSignal(movable_parent, COMSIG_MOVABLE_NEWTONIAN_MOVE, .proc/newtonian_impulse)
```
It just bugged me that a signal hooked to ``movable_parent`` is squashed between two hooked to ``drifting_loop``.

Reply:
yes honey